Source/WebKit/Modules/OSX_Private.modulemap

```diff
@@ -2014,6 +2014,11 @@ framework module WebKit_Private [system] {
     export *
   }
 
+  explicit module _WKTextRun {
```

Comment:
I think we should stop trying to add any new explicit modules to the private module, since it’s not designed to have a sub module for each type, and also causes greater confusion since we have both the general internal module and a bunch of random sub modules